content/en/blog/how-to-use-fission-functions-with-postgresql-database/index.md

```diff
@@ -0,0 +1,357 @@
++++
+title = "How to use Fission functions with PostgreSQL database"
```

Comment:
I think we are using PG with Fission. This sounds more appropriate to me.

```
How to use PostgreSQL database with Fission functions
```

Reply:
updated